src/node_quic_session.h

```diff
@@ -876,8 +878,6 @@ class QuicSession : public AsyncWrap,
 
   AliasedFloat64Array state_;
 
-  ngtcp2_mem alloc_info_;
```

Comment:
Fwiw, I think these re-ordering changes shouldn’t affect anything in practice, but I *think* keeping these where they are could otherwise result in undefined behaviour. This is basically just making sure.

Reply:
Makes perfect sense